src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      if (b === 0) {
+        return a;
```

Comment:
In the `divide` method, returning `a` when `b` is zero is not a standard way to handle division by zero. Consider throwing an error or returning a value like `Infinity` to indicate an undefined operation.